source/includes/products/_how-tos.md

```diff
@@ -1,160 +0,0 @@
-## How To: Search Using 'or'
-```shell
-curl 'https://api.bestbuy.com/v1/products(longDescription=iPhone*|sku=7619002)?show=sku,name&pageSize=15&page=5&apiKey=YourAPIKey&format=json'
-```
-
-```javascript
-var bby = require('bestbuy')('YourAPIKey');
-bby.products('longDescription=iPhone*|sku=7619002',{show:'sku,name',pageSize:15,page:5}).then(function(data){
-  console.log(data);
-});
-```
-
-```json-doc
-{
-  "from": 61,
-  "to": 75,
-  "total": 2753,
-  "currentPage": 5,
-  "totalPages": 184,
-  "queryTime": "0.010",
-  "totalTime": "0.045",
-  "partial": false,
-  "canonicalUrl": "/v1/products(longDescription=\"iPhone*\"|sku=7619002)?show=sku,name&page=5&format=json&apiKey=YourAPIKey",
-  "products": [
-    {
-      "sku": 1752654,
-      "name": "Apple - iPhone 4s 8GB Cell Phone - White (AT&T)"
-    },
-    {
-      "sku": 1761045,
-      "name": "Apple - iPhone 4s 8GB Cell Phone - White (Verizon Wireless)"
-    },
-    {
-      "sku": 1729354,
-      "name": "Apple - iPhone 5c 16GB  Cell Phone - Green (AT&T)"
-    },
-    {
-      "sku": 1752291,
-      "name": "Apple - iPhone 5c 16GB Cell Phone - Blue (AT&T)"
-    }
-  ]
-}
-```
-
-There are several description attributes by which you can search, including `longDescription`, `shortDescription`, `description` or `name`. There is a single `SKU` attribute to search based on `SKU`.
-
-In the example below we are searching the `longDescription` for iPhone&#42;. We have appended iPhone with a wildcard `*` so we can search for iPhones with any suffix. We are also looking for any products that have a SKU with a value of **7619002** - note the **or** `|`. Finally, in our example we have updated the number of results that can be returned per page to **15**. Our search will return page **5** of the total **184** pages. Additional information on how to specify the number of results that should be returned per page and which page to return can be found in our Pagination section.
-
-## How To: Search on Reviews
```

Comment:
A comment to ponder: this title just says, "Search on Reviews." Knowing that we have a Reviews endpoint, I might expect the How To to be about that vs. review data in the Products endpoint. Can we clarify that How To heading? (Maybe "How To: Search Based on Review Attributes" or "...based on Reviews Criteria")